app/_components/SelectButton.tsx

```diff
@@ -0,0 +1,42 @@
+import clsx from 'clsx';
+import { FC, SVGProps } from 'react';
+
+import styles from './SelectButton.module.scss';
+
+interface SelectButtonProps {
+  variant: 'simple' | 'detailed'; // 버튼 스타일 구분
+  title: string;
+  description?: string;
+  subTitles?: string[];
+  Icon?: FC<SVGProps<SVGSVGElement>>; // SVG 컴포넌트 타입
+  isSelected?: boolean;
+  onClick?: () => void;
+}
+
+export default function SelectButton({
+  variant = 'simple',
+  title,
+  description,
+  subTitles,
```

Comment:
여기에 default로 빈 배열을 넣으면 밑에서 nullish 체크를 하지 않아도 될 것 같아요

```suggestion
  subTitles = [],
```

Reply:
옷!